### PR TITLE
MAKE-934: support test results with measurements other than ops_per_sec

### PR DESCRIPTION
**Pull request by ancostas**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -52,7 +52,7 @@
         'PyYAML ~= 5.1',
         'requests ~= 2.22.0',
         'structlog ~= 19.1.0',
-        'tenacity ~= 5.0.4',
+        'tenacity ~= 5.0.4'
```
Comment (dbradf) [inline setup.py]: Why this change? I like having the extra `,` so that you don't have to remember to add it if you add a new requirement below this one.

File: src/evergreen/performance_results.py [hunk 2]
```diff
@@ -35,16 +36,11 @@ def __init__(self, test_result, api):
...
-        # field for microbenchmark points in order to query on 'workload'.
-        self.maximum_thread_level, self.maximum_operations_per_second = _get_max_ops_per_sec(
-            test_result)
 
     @property
     def test_results(self):
-        if self.maximum_operations_per_second is None:
-            return None
-        return _get_performance_results(self.json, self._api)
+        return [PerformanceTestResult(item, self._api) for item in
```
Comment (dbradf) [inline src/evergreen/performance_results.py]: [nit] I'm not a huge fan of a line break in the middle of the `for` statement. I find the following easier to read:
```
return [PerformanceTestResult(item, self._api)
            for item in _format_performance_results(self.json['results'])]
```

**Comment by ancostas** [comment]
evergreen merge
